### ISSUE TITLE
[Bug]: redis_config can apparently set the `scheduler.version_check` to `rand-d` in certain scenarios, which gets skipped during startup checks because it is set.

### ISSUE THREAD

**Issue by lamusmaser** [tubearchivist/tubearchivist#611]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read the [how to open an issue](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#how-to-open-an-issue) guide, particularly the [bug report](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#bug-report) section.

### Operating System

Linux-Kubernetes

## Describe the bug
In an attempt to upgrade from v0.4.3 to v0.4.4, a user was unable to start the container. In the user's provided logs, it was indicating that one of the values was failing to load when attempting to pull the "minute" value. Additional troubleshooting found that this was determined to be the cause of `scheduler.version_check` to be set to `rand-d`, instead of the actual values.

## Steps To Reproduce
Unknown what caused the initial value set to `rand-d`, but it seems to be a load-in of the default config without processing for `rand-d`.

## Expected behavior
Handle the `rand-d` value, even if it is set on the `redis_config` side.

### Relevant log output

```shell
N/A - see channel for log images
```

### Anything else?

Details of troubleshooting and resolution from here: https://discord.com/channels/920056098122248193/1176560078506180680

**Comment by lamusmaser** [comment]
Resolution statement from Discord:

This starts with the `build_schedule` method. The first method it calls is the `App.Config().load_new_defaults` method:

First, it loads the default config from the default `config.json` file. This includes setting `version_check` to `rand-d` (random default).
Next, it grabs the configuration from redis with the `get_config_redis` method. This connects to redis and pulls in the values as they are, mostly present to either fail the load if redis isn't present or return an empty set if the config is not set in redis, yet.
This will then check for customizations - if they are unset, then it will set those configurations (we are going to skip this since it doesn't apply).
Then, for each key/value pair in the `default_config`, it will check if it exists in the `redis_config`. If it is missing, then it will push the default pair.
If the key has nested key/value pairs, it will iterate through those. However, this is where it is getting skipped. It is only checking if the `sub_key` doesn't already exist in the `redis_config`, which it does for this item. Normally, if it was missing, it would then detect the `rand-d`, set it properly, then you would be good to go. Instead, it skips it.

This seems to be the section causing the problems:
https://github.com/tubearchivist/tubearchivist/blob/master/tubearchivist/home/src/ta/config.py#L108-L111

My recommendation would be to delete the `sub-key`, let it generate properly, then we are good to go.
```
docker exec -it Redis redis-cli json.del ta:config $..scheduler.version_check
```
I think something caused the interruption at some point during the config process and it just set it. No idea why, but it should work after that.
I'll submit a fix for this so it shouldn't occur again.

**Comment by bbilly1** [comment]
merged in v0.4.5
